### DISCUSSION TITLE
Svelte in LazyExtra

### DISCUSSION THREAD

**Discussion by Muzammil-hasan** [LazyVim/LazyVim#3405] [Ideas]
Hi Folke,

Thank you once again for adding Astro to LazyVim Extras! Your work is greatly appreciated. If it’s not too much trouble, could you also consider adding Svelte?

Thanks a lot for your help!

**Comment by folke** [comment]
Done!

Most will work already with the new extra, but some features will only work once my mason PR gets merged:

https://github.com/mason-org/mason-registry/pull/5925
